cmd/k8s-operator/deploy/chart/values.yaml

```diff
@@ -4,9 +4,14 @@
 # Operator oauth credentials. If set a Kubernetes Secret with the provided
 # values will be created in the operator namespace. If unset a Secret named
 # operator-oauth must be precreated.
-oauth: {}
+oauth:
   # clientId: ""
+  clientIdKey: client-id
   # clientSecret: ""
+  clientSecretKey: client-secret
+  secret:
+    create: true
+    name: operator-oauth
```

Comment:
This changes the default behaviour from not creating the Secret to creating it and would break everyone who is currently 1) pre-creating the Secret 2) helm installing the operator

Reply:
Oh yes, you are absolutely right.